### ISSUE TITLE
Ingest "Moscow Kamerny Theater" (pudl0113)

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/figgy#1615]
These don't seem to be displayed anywhere. Ingest?

Ping @jpstroop @joycebcat

**Comment by escowles** [comment]
@cwulfman This looks like https://catalog.princeton.edu/catalog/7649172 — Veridian?

**Comment by cwulfman** [comment]
Yes -- Thomas Keenan hand-processed this title a few years ago, but we could never figure out exactly where to put it.

**Comment by cwulfman** [comment]
Let me see if I can find the METS/ALTO for it.
